mungegithub/mungers/submit-queue.go

```diff
@@ -800,6 +800,20 @@ const (
 	unmergeableMilestone    = "Milestone is for a future release and cannot be merged"
 )
 
+// getEarliestApprovedTime expects that lgtmTime and approvedTime are not both
+// nil.  If both of them ARE nil, it will return a nil pointer
+func getEarliestApprovedTime(lgtmTime, approvedTime *time.Time) *time.Time {
```

Comment:
I feel like we should get the lgtmTime and the approvedTime here, in this function...